packages/website/src/bin/handle-config.js

```diff
@@ -35,11 +35,11 @@ const processConfig = (cwd, providedConfig = {}) => {
   let docsList = Array.isArray(docs) ? docs : [docs];
   docsList = docsList.map(doc => {
     const { name, description, path: docPath, urlPath } = doc;
-    const certainUrlPath = urlPath || `./${path.relative(cwd, docPath)}`;
 
     if (!name) throw new Error('name must be provided for all the docs items');
 
     const docsPath = path.resolve(cwd, docPath);
+    const certainUrlPath = urlPath || path.relative(cwd, docsPath);
```

Comment:
I've moved this down so that the urlPath is calculated relative to `docPath` after it's been resolved relative to `cwd`. In practice, cwd is equal to `process.cwd()` so it shouldn't change anything but it fixes an issue in the test where we're passing a mock `cwd`.

I've also removed the prefixed `./`, it doesn't look necessary but maybe I'm missing something @Noviny?

Imo, it might be better to default `urlPath` to the basename of docsPath to prevent URL paths from nesting when docs or the docs.config is not an immediate child of the cwd (root of the repo in most cases).
E.g. for our current dev environment dummy-data project the default urlPath would cause URLs to become 'localhost:3000/dummy-data/simple-project/docs/...' and we have to override it with our own `urlPath` instead.

Reply:
> Imo, it might be better to default urlPath to the basename of docsPath

Yeah, that makes sense to me.

I'll leave it up to you whether to merge this, or update this later and merge this now.

Reply:
Cool, I might merge this now and revisit that change later.